configuration/debian/tedge/postrm

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+set -e
+
+### Remove configs
+
+if [ "$1" = "purge" ]; then
+    rm -r /etc/tedge
```

Comment:
It would be probably good to do rm -rf otherwise it might ask for conformation and leave the directory behind.